nbclean/clean.py

```diff
@@ -132,6 +156,7 @@ def save(self, path_save):
         dir_save = os.path.dirname(path_save)
         if self._verbose is True:
             print('Saving to {}'.format(path_save))
-        if not os.path.isdir(dir_save):
+        # if we are saving to a subdirectory make sure it exists
+        if dir_save and not os.path.exists(dir_save):
```

Comment:
Unrelated to this PR. Got some errors here, but don't remember what exactly. Checking that a path exists feels more natural. I will go and thinking about this change a bit more.